src/algorithms/crypto/vrf.md

```diff
@@ -18,22 +18,14 @@ BLS signature can be used as the basis to construct a VRF. Filecoin transforms
 the BLS signature scheme it uses (see {{<sref signatures Signatures>}} into a
 VRF, Filecoin uses the random oracle model and deterministically hash the
 signature to produce the final digest. Filecoin uses
-[Blake2b-256](https://blake2.net/) (Blake2b with an output size of 256 bits) as
-the hash function. The algorithm is the following:
+SHA256 as the hash function. The algorithm is the following:
 ```
-VRFOutput = Blake2b-256(DST || BLSSignature(message))
+VRFOutput = SHA256(DST || BLSSignature(message))
```

Comment:
Ideally this should be defined in terms of `DeriveRand` also, but I think this can wait for a subsequent PR.

Reply:
It is in all the code.

Reply:
Understood, just flagging that `SHA256(DST || x)` is technically incorrect (should be `SHA256(DST || index || x)`, whether index ends up being 0 or -1 as per discussion above)